app/assets/javascripts/components/dashboard/_dashboard.js.jsx

```diff
@@ -1,18 +1,25 @@
 var Dashboard = React.createClass({
   render(){
-    var user = this.props.user;
-    var projects = this.props.projects;
-    var portfolio = this.props.portfolio;
 
     return (
       <div>
-        <SignedInNavbar user={user}/>
+        <SignedInNavbar user={this.props.user}/>
         <DashboardJumbo/>
           <div className="container dashboard-padding">
             <div className="row dashboard-row">
-              { portfolio != null ? <DashboardProfile user={user} portfolio={portfolio} avatar={this.props.avatar} markdownInfo={this.props.markdownInfo}/> : <DashboardDefaultCard user={user} portfolio={portfolio}/>}
+              { this.props.portfolio != null
+                ?
+                <DashboardProfile user={this.props.user} portfolio={this.props.portfolio} avatar={this.props.avatar} markdownInfo={this.props.markdownInfo}/>
+                :
+                <DashboardDefaultCard user={this.props.user} portfolio={this.props.portfolio}/>
+              }
 
-              { projects != null && projects.length > 0 ?  <DashboardProject projectAvatars={this.props.project_avatars} user={user} projects={projects}/>: <NoDashboardProjects user={user} portfolio={portfolio}/>}
+              { this.props.projects != null && this.props.projects.length > 0
+                ?
+                <DashboardProject projectAvatars={this.props.project_avatars} user={this.props.user} projects={this.props.projects}/>
+                :
+                <NoDashboardProjects user={this.props.user} portfolio={this.props.portfolio}/>
+              }
             </div>
```

Comment:
Not a giant fan of the ternary, but this is a much better way of displaying it that before, so good job on the refactor.